deploy/synology_dsm.sh

```diff
@@ -96,8 +96,15 @@ synology_dsm_deploy() {
   if [ -n "$SYNO_TOTP_SECRET" ]; then
     if _exists oathtool; then
       otp_code="$(oathtool --base32 --totp "${SYNO_TOTP_SECRET}" 2>/dev/null)"
+    elif _exists docker; then
+      if [[ "$(docker images -q toolbelt/oathtool:latest 2> /dev/null)" == "" ]]; then
+        read -e -p "docker is available but oathtool docker image must be downloaded, do you want to download it (Y) or abort (N) ? " choice
```

Comment:
don't use this interactive anwsers.
just show an info message ,and use the  "docker run"  command.

Reply:
@tresni requested a prompt before pulling the docker image... do I added the interactive answer.

But I can remove the prompt and directly pull the image, it was my first attempt before @tresni's request.

I don't know how to comply with tresni's request but not requesting user interaction...

Reply:
I think the answer is to display the info message and tell the user to install the docker image using `docker pull`.  Pulling down a container automatically into an unknown environment is a bad idea. So if @Neilpage is saying no to interactive commands, then the only thing to do is show a message and exit.

Reply:
I think we need to put an option around the docker stuff anyway as `docker run` will pull an image if it's not available locally. So this whole block likely needs to change.

```
$ docker run --rm -it toolbelt/oathtool --base32 --totp BADD0G
Unable to find image 'toolbelt/oathtool:latest' locally
latest: Pulling from toolbelt/oathtool
8921db27df28: Pull complete
882084980d3c: Pull complete
Digest: sha256:f4c7fafb00fef5178ffbdf7b6e9f58e234617bb035967c3ba6f879f1f47a4bf3
Status: Downloaded newer image for toolbelt/oathtool:latest
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
oathtool: base32 decoding failed: Base32 string is invalid
```

Reply:
Works for me!